wordpress/README.md

```diff
@@ -0,0 +1,280 @@
+
+If you get disconnected from the terminal, you can reconnect to the installer                                                                                                                                                                   
+agent by issuing 'gravity resume' command.                                                                                                                                                                                                      
+
+If the installation fails, use 'gravity plan' to inspect the state and                                                                                                                                                                          
+'gravity resume' to continue the operation.                                                                                                                                                                                                     
+See https://gravitational.com/gravity/docs/cluster/#managing-an-ongoing-operation for details.                                                                                                                                                  
+
+Sun Apr 26 23:55:11 UTC Connecting to installer
+Sun Apr 26 23:55:32 UTC Connected to installer
+Sun Apr 26 23:55:32 UTC Successfully added "master" node on 10.150.15.236
+...
+
+Mon Apr 27 00:03:05 UTC Install application wordpress:0.0.1
+Mon Apr 27 00:03:05 UTC Executing install hook for wordpress:0.0.1
+Mon Apr 27 00:03:15 UTC         Still executing install hook for wordpress:0.0.1 (10 seconds elapsed)
+Mon Apr 27 00:03:16 UTC Executing "/connect-installer" locally
+Mon Apr 27 00:03:17 UTC Connecting to installer
+Mon Apr 27 00:03:17 UTC Connect to installer
+Mon Apr 27 00:03:19 UTC Executing "/election" locally
+Mon Apr 27 00:03:19 UTC Enable leader elections
+Mon Apr 27 00:03:19 UTC Enable cluster leader elections
+Mon Apr 27 00:03:20 UTC Executing operation finished in 6 minutes
+Mon Apr 27 00:03:20 UTC The operation has finished successfully in 7m48s
+Mon Apr 27 00:03:21 UTC
+Cluster endpoints:
+    * Authentication gateway:
+        - 10.150.15.236:32009
+    * Cluster management URL:
+        - https://10.150.15.236:32009
+
+Application endpoints:
+    * wordpress:0.0.1:
+        - wordpress:
+            - http://None:3306
+
+```
+
+**Congratulations!** You have created a fully functional Kubernetes cluster
+with Wordpress running inside. To check the health and status of the Cluster,
+execute this command on the target node:
+
+```
+$ sudo gravity status
+Cluster name:           wordpress
+Cluster status:         active
+Cluster image:          wordpress, version 0.0.1
+Gravity version:        7.0.1 (client) / 7.0.` (server)
```

Comment:
Is the backtick actually in the output or is it a typo?

Reply:
User keyboard error.  fixed.